config/webpack.config.js

```diff
@@ -52,11 +50,16 @@ var config = {
       path.join(__dirname, '..', 'webpack'),
```

Comment:
Do you mind making the change @ohadlevy mentions in https://github.com/theforeman/foreman/pull/4733#issuecomment-332848660?

Reply:
in that case :+1: from my side to add those changes. /cc @dLobatog as it seems we would need to find a different approach to package this.

Reply:
It did indeed broke packaging which means our nightlies are now broken. Given the current load on the packaging and release team it might be wise to slow down these changes, at least until 1.16 is out.